Cargo.toml

```diff
@@ -9,6 +9,7 @@ repository = "https://github.com/BurntSushi/aho-corasick"
 readme = "README.md"
 keywords = ["string", "search", "text", "aho", "corasick"]
 license = "Unlicense/MIT"
+exclude = ["benches/sherlock.txt"]
```

Comment:
The rest of it isn't large. Also I would rather not have cargo start complaining due to an incomplete crate if it can't find bench.rs as specified in the toml.